### ISSUE TITLE
why not move to 50 or 60 blocks?

### ISSUE THREAD

**Issue by sysnotdown** [leela-zero/leela-zero#2224]
the progress is very slow now ,  40 blocks may reach the limit. 

so, why not explore 50 or 60 blocks from now on?

**Comment by wonderingabout** [comment]
@sysnotdown 

if 40b is slow, then you can be sure 50 or 60b will be even more slow

also, 40b didnt reach its max potential yet

and finally, considering even alphago zero didnt need to go above 40b, we'd rather want to restart since the begining to improve training settings, rather than train unefficiently a bigger size

**Comment by nerai** [comment]
No opinion on whether we should try larger b, but:

> also, 40b didnt reach its max potential yet
20b did not reach its maximum yet either, but we don't use it to train, because 40b acts more efficient.

**Comment by wonderingabout** [comment]
with leela zero's limited ressource, its gonna be hard

also, again, not efficient seeing how far alphago zero got just with 40b

we want people to play a 60b lz on ogs and other servers, or home use, gonna be very slow (and weaker)
so not efficient i think

**Comment by Marcin1960** [comment]
@nerai "20b did not reach its maximum yet either, but we don't use it to train, because 40b acts more efficient."

If 20b had the resources that are consumed by the slow 40b (plus resources of those with "bad" computers), it would be much more efficient.

**Comment by l1t1** [comment]
bjiyxo has made very strong 15b weights with 40b train data, maybe he also can made a 20b one with same method

**Comment by wonderingabout** [comment]
they talked about not being successful with 20b so far, maybe they want to have a look at it again some time later

**Comment by sysnotdown** [comment]
> max potential yet



> @sysnotdown
> 
> if 40b is slow, then you can be sure 50 or 60b will be even more slow
> 
> also, 40b didnt reach its max potential yet
> 
> and finally, considering even alphago zero didnt need to go above 40b, we'd rather want to restart since the begining to improve training settings, rather than train unefficiently a bigger size

we don't have to push 40b to its max power. that may not so efficient.  more blocks may get higher elo with less computation power.

**Comment by sysnotdown** [comment]
> bjiyxo has made very strong 15b weights with 40b train data, maybe he also can made a 20b one with same method

that's what I want to say!

**Comment by sysnotdown** [comment]
> No opinion on whether we should try larger b, but:
> 
> > also, 40b didnt reach its max potential yet
> 
> 20b did not reach its maximum yet either, but we don't use it to train, because 40b acts more efficient.

yes! you get it!

**Comment by wonderingabout** [comment]
i think this is a wrong direction

we want to increase efficiency in the training = max learning for minimal ressource

what you say is you dont care about ressource, only want fast improvement ??

then i ask you this ?
how can you improve fast if you dont care about ressource ??

then, if you have ressource, you want to make the best AI, not a fast to improve AI with many problems, see what minigo or elf team is doing with their big ressource

**Comment by sysnotdown** [comment]
> i think this is a wrong direction
> 
> we want to increase efficiency in the training = max learning for minimal ressource
> 
> what you say is you dont care about ressource, only want fast improvement ??
> 
> then i ask you this ?
> how can you improve fast if you dont care about ressource ??
> 
> then, if you have ressource, you want to make the best AI, not a fast to improve AI with many problems, see what minigo or elf team is doing with their big ressource

I think 40b is obvious close to its limit.  continue trainning 40b is not efficient now.
if we can try 50b, maybe we can find more potencial up-going space.  
50b will cost much more resources than 40b? I don't think so.  computer will get faster and faster, we should not worry too much about it.
if we can't try 60b, I suggest we try 50b at first, and see its potencial space.  40b is running out its power now.

**Comment by wonderingabout** [comment]
the problem is not 40b limit, but the problem is leela zero limit in 40b

leela zero can improve the 40b training system, but this is not easy to do, and too late already so far in the training

**Comment by wonderingabout** [comment]
moving to 50b or 60b will make the same problem

**Comment by Marcin1960** [comment]
l1t1: "bjiyxo has made very strong 15b weights with 40b train data, maybe he also can made a 20b one with same method"

Better yet; use 40b train data for the start, like it was done with ELF data, and then let 20b train on its own, faster and focusing on diversity.

More, perhaps 17b or 18b will be sufficient to do the same? Could be interesting to verify.

**Comment by Marcin1960** [comment]
@sysnotdown:  "I think 40b is obvious close to its limit"

No, the opposite is obvious. The size is not the limit, the size is already more than needed. Increasing it even more will bog the project down.

**Comment by l1t1** [comment]
good idea should come with proof, just like elf,minigo and bjiyxo did. if we havent that,  just use their works is better

**Comment by jillybob** [comment]
Best way to improve going forward is using the distillation method. Use 40b net, to train a smaller net (25 or 30 layer net). Then use this smaller and faster (than 40b) strong net to generate games. Then train a 40b or 50b using the games generated by the 25 or 30b net. 

The aim of this is to generate a 25 or 30b net, with 40b games, so that it is stronger than the 40b at time parity. This will lead to more games generated per day than the 40b net, that we can use to create a strong 40 or 50b.
